app/Http/Middleware/Admin.php

```diff
@@ -22,6 +22,6 @@ public function handle(Request $request, Closure $next)
             return $next($request);
         }
 
-        abort(403, 'Access denied');
+        return redirect('/');
```

Comment:
perhaps you could also have a message to be displayed? Like "Oops! It seems like you do not have the right permissions to do this." hehe